code/modules/client/client_procs.dm

```diff
@@ -325,6 +325,14 @@
 	var/sql_computerid = sql_sanitize_text(src.computer_id)
 	var/sql_admin_rank = sql_sanitize_text(admin_rank)
 
+	if ((player_age == -1) && !(ckey in global.panic_bunker_bypass)) //first connection
```

Comment:
It seems this branch will never be traversed, as `player_age` is set to `0` on line 285 (unless the db returns an explicit `-1`, which also seems impossible).